src/webgpu/api/validation/resource_usages/buffer/in_pass_encoder.spec.ts

```diff
@@ -231,6 +231,11 @@ referenced by that bind group is "used" in the usage scope. `
         ) {
           return false;
         }
+
```

Comment:
In the test description for both of these tests, add a link to the issue that tracks the new tests (doesn't need to be a TODO, just something so there's a followable reference to the new tests).